XenAdmin/Wizards/CrossPoolMigrateWizard/CrossPoolMigrateDestinationPage.cs

```diff
@@ -64,11 +64,6 @@ public CrossPoolMigrateDestinationPage(Host preSelectedHost, List<VM> selectedVM
             InitializeText();
         }
 
-        public override bool EnableNext()

```

Comment:
With this change, when moving from this page to previous page and then back again, the Destination selection is lost (this is an existing issue, not caused by these changes), but the Next button remains enabled and I can proceed to the Next page.

Reply:
The latest changes improve slightly the behaviour by restoring selection after moving backwards from and then forwards to the destination page at pool level - not the host mapping though; this requires more changes which I think have started going outside the scope of this PR.